servers/physics_server_3d.h

```diff
@@ -468,7 +479,6 @@ class PhysicsServer3D : public Object {
 	virtual void body_set_force_integration_callback(RID p_body, Object *p_receiver, const StringName &p_method, const Variant &p_udata = Variant()) = 0;
 
 	virtual void body_set_ray_pickable(RID p_body, bool p_enable) = 0;
-	virtual bool body_is_ray_pickable(RID p_body) const = 0;
```

Comment:
Why is this method removed (same with area_is_ray_pickable)? It looks like it could be still supported.

Reply:
It wasnt in the other sever when I copied, but the idea is to remove most of the query methods that return values from the servers (except for shape data and body state), since they serve no purpose and they are dangerous if misused (force thread stall when running the server on a thread). 

I did not go that far in this PR but it should be one of the things to do once merged. I already did this in RenderingServer for the same reason.